### PR TITLE
feat (IBM MQ) : Adding pack for New Relic IBM MQ Monitor that allows monitoring the performance of IBM MQ Objects 

### PR DESCRIPTION
**Pull request by hsinghkalsi**
## Summary

Adding  New Relic IBM MQ Monitor that allows monitoring the performance of IBM MQ Objects like channels, topics and queues

### Screenshots
ibmmq

### CODE REVIEW

**Comment by hsinghkalsi** [comment]
@psomareddy

File: packs/ibmmq/config.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+id: 924fd4b3-a6d1-4a6e-9e2c-b598f197f713
+
+# Name of the pack (required)
+name: ibmmq
+title: New Relic IBM MQ Monitor
+description: >-
```
Comment (polfliet) [inline packs/ibmmq/config.yml:6]: @hsinghkalsi we just found an issue with string parsing
can you please replace all `>-` with `|`?

File: packs/ibmmq/dashboards/ibmmq.json [hunk 2]
```diff
@@ -0,0 +1,224 @@
+{
+  "name": "IBM MQ PROD",
+  "description": null,
```
Comment (polfliet) [inline packs/ibmmq/dashboards/ibmmq.json:3]: Can you please update all description fields to empty strings `""`, so the schema checks don't fail.
We are working to improve this but we unfortunately have to do this manually now

**Review by polfliet — CHANGES_REQUESTED** [review]
Keep 'em coming! Just requesting 2 small fixes for issues we have been running against
Thanks!

File: packs/ibmmq/config.yml (hunk 1 above)
Comment (hsinghkalsi) [reply packs/ibmmq/config.yml]: Done!

File: packs/ibmmq/dashboards/ibmmq.json (hunk 2 above)
Comment (hsinghkalsi) [reply packs/ibmmq/dashboards/ibmmq.json]: Fixed! @polfliet

File: packs/ibmmq/config.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+# Authors of the pack (required)
+authors:
+  - New Relic Labs
+
+documentation:
+  - name: New Relic integration for IBM MQ 
+    description: |
+      The New Relic IBM MQ Monitor enables performance monitoring of IBM MQ Objects like channels, topics and queues.
+    url: https://github.com/newrelic-experimental/newrelic-ibm-mq-monitor
```
Comment (jpvajda) [inline packs/ibmmq/config.yml:35]:
```suggestion
    url: >-
    https://github.com/newrelic-experimental/newrelic-ibm-mq-monitor
```

Comment (jpvajda) [reply packs/ibmmq/config.yml:35]: see this section of the contributor guide about this change

https://github.com/newrelic/newrelic-observability-packs/blob/main/CONTRIBUTING.md#documentation

Comment (jpvajda) [reply packs/ibmmq/config.yml:35]: you will also need to tab the URL 1 tab.

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: packs/ibmmq/config.yml (hunk 3 above)
Comment (polfliet) [reply packs/ibmmq/config.yml]: I quickly made this update. Most of the existing quickstarts don't follow this url format, so we should run a script to update them

**Review by polfliet — APPROVED** [review]

**Pull request merged**